Gemfile

```diff
@@ -77,6 +77,8 @@ gem 'devise'
 
 gem 'cancancan'
 
```

Comment:
As a result of the wrong order of migration files, I was unable to test your API, the next reviewer will test it and give appropriate feedback. 

I wish you the best.